### PR TITLE
Task/72 - Create onboarding screen

### PR DESCRIPTION
**Pull request by jvrastil**
- when user starts application the onboarding screen is displayed
- currently there are 3 prepared screens (different color, text will be filled later)
- on long press progress is paused
- on tap on left side previous screen is displayed
- on tap on right side next screen is displayed

### CODE REVIEW

File: lib/view/routes/unauthenticated_route.dart [hunk 1]
```diff
@@ -55,4 +66,9 @@ class UnauthenticatedRoute {
         throw const FormatException('Screen not found');
     }
   }
+
+  Future<bool> _getPreferences() async {
```
Comment (storkandstars) [inline lib/view/routes/unauthenticated_route.dart:70]: Remove this functionality for now. It is not necessary to store if a user passed this screen. If this requirement will be raised in future we will implement it then.

Remove, please also related packages.

File: lib/view/widgets/story.dart [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                  ),
+                ),
+              ),
+            )
+            .toList(),
+      ),
+    );
+
+    return GestureDetector(
+      onLongPressStart: (_) => _pauseAnimation(),
```
Comment (storkandstars) [inline lib/view/widgets/story.dart:53]: Consider changing long press event to tap down event so there is no delay before pausing the loop animation

File: lib/view/widgets/story.dart [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    value: controllers[e.key].value,
+                  ),
+                ),
+              ),
+            )
+            .toList(),
+      ),
+    );
+
+    return GestureDetector(
```
[the remark this answers is not in the document]
Comment (jvrastil) [reply lib/view/widgets/story.dart:52]: Agreed, done

File: lib/view/widgets/story.dart (hunk 2 above)
Comment (jvrastil) [reply lib/view/widgets/story.dart:53]: ok

File: lib/view/routes/unauthenticated_route.dart (hunk 1 above)
Comment (jvrastil) [reply lib/view/routes/unauthenticated_route.dart]: done

**Review by storkandstars — APPROVED** [review]

**Pull request merged**